packages/block-library/src/navigation-link/index.php

```diff
@@ -174,7 +103,7 @@ function render_block_core_navigation_link( $attributes, $content, $block ) {
 		return '';
 	}
 
-	$colors          = block_core_navigation_link_build_css_colors( $block->context, $attributes );
+	$colors          = block_core_navigation_submenu_build_css_colors( $block->context, $attributes );
```

Comment:
Is it possible that this function won't be defined? I suppose it is possible for someone to deregister the navigation submenu block, but not the navigation-link block, though I can't think why you would do that. We could move the function to the parent block...